app/models/listing.rb

```diff
@@ -10,9 +10,16 @@ class Listing < ActiveRecord::Base
   has_many :listing_images
   has_many :reservations
   scope :city, -> (city) { where city: city }
-  scope :max_guests, -> (max_guests) { where("max_guests >= ?", max_guests) }
+  scope :max_guests, -> (max_guests) { where("max_guests >= ?", max_guests)
+    .order(max_guests: :asc) }
   scope :category_id, -> (category_id) { where category_id: category_id }
-  scope :nightly_rate, -> (nightly_rate) { where("nightly_rate <= ?", nightly_rate) }
+  scope :nightly_rate, -> (nightly_rate) { where("nightly_rate <= ?", nightly_rate)
```

Comment:
Line is too long. [83/80]<br>Block body expression is on the same line as the block start.